Wikipedia/Code/WMFFeedArticlePreview.m

```diff
@@ -44,13 +44,10 @@ + (NSValueTransformer *)thumbnailURLJSONTransformer {
 
 + (NSValueTransformer *)articleURLJSONTransformer {
     return [MTLValueTransformer
-        transformerUsingForwardBlock:^NSURL *(NSDictionary *value,
+        transformerUsingForwardBlock:^NSURL *(NSString *value,
                                               BOOL *success,
                                               NSError *__autoreleasing *error) {
-            NSString *lang = value[@"lang"];
-            NSString *normalizedTitle = value[@"normalizedtitle"];
-            NSURL *siteURL = [NSURL wmf_URLWithDefaultSiteAndlanguage:lang];
-            return [siteURL wmf_URLWithTitle:normalizedTitle];
+            return [NSURL URLWithString:value];
         }
         reverseBlock:^NSDictionary *(NSURL *articleURL,
                                      BOOL *success,
```

Comment:
The reverse block needs to be updated to return the URL's absolute string instead of the dictionary. Also, `+ (NSUInteger)modelVersion` should be bumped to 4